css/_components.scss

```diff
@@ -39,3 +39,23 @@
         }
     }
 }
+
+/**
+ * The header mixin works with js (sticky-header.js) to hide the header as you scroll down, and show it as you 
+ * scroll back up. This is responsive as the "top" value is dynamically set in the js based on the height of the
+ * header. Top padding is also added to the body dynamically via js so that the header doesn't overlap any
+ * content. 
+ * 
+ * Markup requires: header.primary
+ *
+ **/
+
+@mixin header() {
```

Comment:
Should use a more descriptive name for the mixin than just 'header' - maybe 'sticky-header'